Lib/glyphsLib/glyphdata.py

```diff
@@ -257,11 +259,11 @@ def _translate_category(glyph_name, unicode_category):
         "Cf": ("Separator", "Format"),
         "Cn": ("Symbol", None),
         "Co": ("Letter", "Compatibility"),
-        "Ll": ("Letter", "Lowercase"),
+        "Ll": ("Letter", None),
         "Lm": ("Letter", "Modifier"),
         "Lo": ("Letter", None),
-        "Lt": ("Letter", "Uppercase"),
-        "Lu": ("Letter", "Uppercase"),
+        "Lt": ("Letter", None),
+        "Lu": ("Letter", None),
```

Comment:
@schriftgestalt why this change? What does it have to do with updating the GlyphData database?

Reply:
I see you commented on this already

> The case information moved from subcategory="Upper/Lowercase" to case="upper/lower". Not sure if that is used anywhere.

If it were me, i'd just have left it as it was. Let's hope nothing else breaks

Reply:
But it was failing some tests.